docs/standard/getting-started.md

```diff
@@ -1,18 +1,36 @@
 ---
 title: Getting Started
 description: Getting started with .NET
-keywords: .NET, .NET Core
+keywords: .NET
 author: cartermp
 manager: wpickett
-ms.date: 10/17/2016
+ms.author: phcart
+ms.date: 11/16/2016
 ms.topic: article
-ms.prod: .net-core
+ms.prod: .net
 ms.technology: .net-core-technologies
 ms.devlang: dotnet
-ms.assetid: bbfe6465-329d-4982-869d-472e7ef85d93
+ms.assetid: 81c07080-acdf-4aef-a66d-0ab52fab2c04
 ---
 
 # Getting Started
 
-> [!NOTE]
-This article is in-progress.
\ No newline at end of file
+There are a number of ways to get started with .NET.  Because .NET is a large and massive platform, there are multiple articles in this documentation which show how you can get started with .NET, each from a different perspective.
+
+## Getting started using .NET languages
+
+* For C#, the [Getting Started](../csharp/getting-started/index.md) articles and [C# Tutorials](../csharp/tutorials/index.md) provide a number of ways to get started in a C#-centric way.
```

Comment:
Since all three of these have the same link text, I'd re-write them something like:

"The [C# Getting Started](../csharp/getting-started/index.md) and [C# Tutorials](../csharp/tutorials/index.md) provide a number of ways to get started in a C# centric way."

similarly with F# and VB.NET